pkg/securityscan/scanHandler.go

```diff
@@ -293,8 +293,14 @@ func (c Controller) setClusterScanStatusDisplay(scan *v1.ClusterScan) {
 			display.Message = "ClusterScan complete, there are some test failures, please check the ClusterScanReport"
 			display.Error = true
 		} else {
-			display.State = passedState
-			display.Error = false
+			if summary.Warn > 0 && scan.Spec.ScoreWarning == v1.ClusterScanFailOnWarning {
```

Comment:
Where are we setting the summary.Warn value? It should be set by the jobHandler

Reply:
ah yes, while adding this and testing the changes, I was vendoring in [this commit](https://github.com/rancher/security-scan/pull/30/files#diff-cdbf6a6affc59d13e5fa37a2e5a29596d90e108f7bf5cc20a29e7d0929d56069R62) that adds the "Warn" field on Report from report.go in secruity-scan.
And then I was setting it in [the jobHandler](https://github.com/rancher/cis-operator/pull/53/files#diff-2d0084fb0dcb7b80b1a369ead00ce999546aaff3ca3ecfa430dd7b990eae96bbR150)
I think I missed out this line while separating out the commit, good catch!
Right now I can't add it, because the security-scan Report type currently in vendor does not have it. I can still add it in the PR but build will fail so I think we need to merge the security-scan PR first so I can vendor it in https://github.com/rancher/security-scan/pull/30

Reply:
accidentally hit "request review" again. Will add in that line along with vendor changes and re-request review

Reply:
added it in along with vendored changes